doc/api/cards/filter_-_card_search_operator.json

```diff
@@ -0,0 +1,44 @@
st_22_09`\n* **snapshot**: Type: array\n  * `snapshot_id` of a snapshot containing the card for a printing.\n* **universal_faction_cost**: Type: array\n  * Concatenation of `restriction_id` and a Universal Faction Cost value, joined by a hyphen, like `universal_faction_cost:napd_mwl_1_2-3`.\n* **additional_cost**: Type: boolean\n  * Does the card text for a printing specify an additional cost to play?\n* **advanceable**: Type: boolean\n  * Is the card for a printing advanceable?\n* **gains_subroutines**: Type: boolean\n  * Does the card for a printing text allow for adding or gaining subroutines?\n* **in_restriction**: Type: boolean\n  * Is the card for a printing specified on any Restriction list?\n* **interrupt**: Type: boolean\n  * Does the card for a printing have an interrupt ability?\n* **is_unique, u**: Type: boolean\n  * Is the card for a printing unique?\n* **on_encounter_effect**: Type: boolean\n  * Does the card text for a printing specify an on encounter effect?\n* **performs_trace**: Type: boolean\n  * Does the card for a printing perform a trace?\n* **rez_effect**: Type: boolean\n  * Does the card have a rez effect?\n* **trash_ability**: Type: boolean\n  * Does the card for a printing provide a trash ability?\n* **advancement_cost, g**: Type: integer\n  * The `advancement_cost` value for an agenda. Accepts positive integers and X (case-insensitive).\n* **agenda_points, v**: Type: integer\n  * The printed number of agenda points for the agenda.\n* **base_link, l**: Type: integer\n  * The printed link value for an Identity.\n* **cost, o**: Type: integer\n  * The printed cost of the card for a printing.\n* **influence_cost, n**: Type: integer\n  * The influence cost or number of influence pips for the card for a printing.\n* **link_provided**: Type: integer\n  * The amount of link provided.\n* **memory_usage, m**: Type: integer\n  * The memory (MU) cost of this card.\n* **mu_provided**: Type: integer\n  * The amount of memory (MU) provided by the card for a printing.\n* **num_printed_subroutines**: Type: integer\n  * The number of printed subroutines on this card.\n* **num_printings**: Type: integer\n  * Count of unique printings for this card.\n* **recurring_credits_provided**: Type: integer\n  * The number of recurring credits provided by this card.  Accepts integers or X.\n* **strength, p**: Type: integer\n  * The strength of the card for a printing.\n* **trash_cost, h**: Type: integer\n  * The trash cost of this card.\n* **attribution**: Type: string\n  * The designer of this card text, if specified.\n* **card_type, t**: Type: string\n  * `card_type_id` of this card.\n* **faction, f**: Type: string\n  * `faction_id` of this card.\n* **side, d**: Type: string\n  * `side_id` of the card for a printing.\n* **text, x**: Type: string\n  * The text of a card for a printing, stripped of all formatting symbols and marks.\n* **title, _**: Type: string\n  * The title of a card for a printing, stripped of all formatting symbols and marks.\n",
```

Comment:
> **Boolean** - supports the `:` (match) and `!` (negated match) operators. `true`, `false`, `t`, `f``, `1`', and `0`` are all acceptable values.
This doesn't render properly (neither does the printing equivalent).

Reply:
oops!  Good catch.  Thanks!